diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index a573819..7e10c5c 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -363,7 +363,8 @@ dtb-$(CONFIG_SOC_VF610) += \
 	vf500-colibri-dual-eth.dtb \
 	vf610-colibri-dual-eth.dtb \
 	vf610-cosmic.dtb \
-	vf610-twr.dtb
+	vf610-twr.dtb \
+	ipeboard.dtb
 dtb-$(CONFIG_ARCH_MXS) += \
 	imx23-evk.dtb \
 	imx23-olinuxino.dtb \
